### DISCUSSION TITLE
Next Js Next Auth Login is not working in vercel platform. (It's working local not wroking in vercel)

### DISCUSSION THREAD

**Discussion by IndraDaman** [vercel/community#3747] [Help] [answered]
### Summary

I use next js react framework with next auth for login and authentication. My project run perfectly in local environment even I build and then run my project locally it doesn't have any issue.

But When I upload it into the vercel it shows only login page and when I try to login it just display error and that's it. It doesn't show any error details or anything else.

I am stuck into this situation please help and resolve my issue. If you require any furthur details please feel free to reach me

### Example

https://orchid-vercel-test1.vercel.app/

### Steps to Reproduce

I use next js react framework with next auth for login and authentication. My project run perfectly in local environment even I build and then run my project locally it doesn't have any issue.

**Comment by amyegan** [comment]
Duplicate of https://github.com/vercel/community/discussions/3754

**Reply by IndraDaman** [reply]
Yes I add environment variables in project setting alsow add base url next auth url as vercle current url. Still getting this error.

**Comment by IndraDaman** [comment, marked as answer]
Finally got solution, I don't know why but in next auth [...nextauth].ts not support api calling method
fetch(base_Url + "", { method: "POST", headers: { "Content-Type": "application/json", }, body: JSON.stringify({ userName: email, password: password, }), });
Is not working so I am using Axios method for calling api.
axios.post(base_Url+"", JSON.stringify({ userName: email, password: password, }),{headers: { "Content-Type": "application/json", }});

And it is working perfectly.
